musicbot/bot.py

```diff
@@ -1482,7 +1493,12 @@ def check(m):
 
         except ValueError:
             raise exceptions.CommandError('{} is not a valid number'.format(new_volume), expire_in=20)
-
+			
+        if permissions.max_volume and new_volume > permissions.max_volume:
+            raise exceptions.PermissionsError(
+                "[ERROR] You are not able to set the volume above (%s)" % permissions.max_volume
```

Comment:
What if i was to have the [Error]    is they have permissions to change it but try going above the set limit, and the default if they don't ?